nestkernel/connector_model.h

```diff
@@ -103,7 +103,7 @@ class ConnectorModel
 
   virtual void set_syn_id( synindex syn_id ) = 0;
 
-  virtual std::vector< SecondaryEvent* > create_event( size_t n ) const = 0;
+  virtual SecondaryEvent* create_event() const = 0;
```

Comment:
Why no longer a vector?

Reply:
Because only element 0 of the vector was ever used. I assume this should have been refactored a long time ago...